centreon-open-tickets/www/modules/centreon-open-tickets/providers/Ivanti/IvantiProvider.class.php

```
@@ -0,0 +1,838 @@
+    * check if the close option is enabled, if so, try to close every selected ticket
+    *
+    * @param {array} $tickets
+    *
+    * @return void
+    */
+    public function closeTicket(&$tickets): void
+    {
+        if ($this->doCloseTicket()) {
+            foreach ($tickets as $k => $v) {
+                try {
+                    $ticketId = $v['ticket_id'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$ticketId'. (undefined)

(UnusedLocalVariable)